versions.json

```diff
@@ -1,5 +1,5 @@
 [
   "1.2",
   "1.1",
-  "1.3.0-preview"
+  "1.3.0"
```

Comment:
"1.3"  
"1.2",
"1.1"

Reply:
On this topic, I was thinking the next minor release should be `1.4-preview` rather than `1.4.0-preview`